packages/js/src/classic-editor/helpers/dom.js

```diff
@@ -5,173 +5,352 @@ import { excerptFromContent } from "../../helpers/replacementVariableHelpers";
+export const getPostContent = () => getContentTinyMce( DOM_IDS.POST_CONTENT ) || "";
 
 /**
- * Gets the date from the document.
+ * Gets the term description from the document.
  *
- * @returns {string} The date or an empty string.
+ * @returns {string} The term description or an empty string.
  */
-export const getDate = () => `${ getDateYear() }-${ getDateMonth() }-${ getDateDay() }`;
+export const getTermDescription = () => getContentTinyMce( DOM_IDS.TERM_DESCRIPTION ) || "";
 
 /**
- * Gets the meta description from the document.
+ * Gets the post date month from the document.
  *
- * @returns {string} The meta description or an empty string.
+ * @returns {string} The post date month or an empty string.
  */
-export const getMetaDescription = () => get( document.getElementById( DOM_YOAST_IDS.META_DESCRIPTION ), "value", "" );
+export const getPostDateMonth = createGetDomElementProp( DOM_IDS.POST_DATE_MONTH );
 
 /**
- * Gets the SEO title from the document.
+ * Gets the post date day from the document.
  *
- * @returns {string} The SEO title or an empty string.
+ * @returns {string} The post date day or an empty string.
  */
-export const getSeoTitle = () => get( document.getElementById( DOM_YOAST_IDS.SEO_TITLE ), "value", "" );
+export const getPostDateDay = createGetDomElementProp( DOM_IDS.POST_DATE_DAY );
 
+/**
+ * Gets the post date year from the document.
+ *
+ * @returns {string} The post date year or an empty string.
+ */
+export const getPostDateYear = createGetDomElementProp( DOM_IDS.POST_DATE_YEAR );
+
+/**
+ * Gets the post date from the document.
+ *
+ * @returns {string} The post date or an empty string.
+ */
+export const getPostDate = () => `${ getPostDateYear() }-${ getPostDateMonth() }-${ getPostDateDay() }`;
+
+/**
+ * Gets the post SEO title from the document.
+ *
+ * @returns {string} The post SEO title or an empty string.
+ */
+export const getPostSeoTitle = createGetDomElementProp( DOM_YOAST_IDS.POST_SEO_TITLE );
+
+/**
+ * Gets the term SEO title from the document.
+ *
+ * @returns {string} The term SEO title or an empty string.
+ */
+export const getTermSeoTitle = createGetDomElementProp( DOM_YOAST_IDS.TERM_SEO_TITLE );
+
+/**
+ * Gets the post meta description from the document.
+ *
+ * @returns {string} The post meta description or an empty string.
+ */
+export const getPostMetaDescription = createGetDomElementProp( DOM_YOAST_IDS.POST_META_DESCRIPTION );
+
+/**
+ * Gets the term meta description from the document.
+ *
+ * @returns {string} The term meta description or an empty string.
+ */
+export const getTermMetaDescription = createGetDomElementProp( DOM_YOAST_IDS.TERM_META_DESCRIPTION );
+
+/**
+ * Gets the term focus keyphrase from the document.
+ *
+ * @returns {string} The term focus keyphrase or an empty string.
+ */
+export const getPostIsCornerstone = () => isEqual( "1", get( document.getElementById( DOM_YOAST_IDS.POST_IS_CORNERSTONE ), "value", "0" ) );
```

Comment:
I'd prefer the variable before the hard value in a comparison like this. With if statements this is our codestyle.

```suggestion
export const getPostIsCornerstone = () => isEqual( get( document.getElementById( DOM_YOAST_IDS.POST_IS_CORNERSTONE ), "value", "0" ), "1" );
```